cmake/utilities.cmake

```diff
@@ -644,3 +644,28 @@ function(get_fprime_library_option_string OUTPUT_VAR LIBRARY_NAME)
     string(REGEX REPLACE "[^A-Z0-9_]" "_" LIBRARY_OPTION "${LIBRARY_NAME_UPPER}")
     set("${OUTPUT_VAR}" "${LIBRARY_OPTION}" PARENT_SCOPE)
 endfunction(get_fprime_library_option_string)
+
+####
+# Function `resolve_path_variables`:
+#
+# Resolve paths updating parent scope.  ARGN should contain a list of variables to update.
+#
+# ARGN: list of variables to update
+####
+function(resolve_path_variables)
```

Comment:
Throughout the PR, `*_RESOLVED` variables are defined. Why not just pass the regular variable in so that it gets resolved? Are the **un**resolved variables needed for some reason?

Reply:
The pre-resolved variables are CMake built-ins.  I do not wish to overwrite them because of the unanticipated consequences of redefining variables that CMake developers expect.

Reply:
got it - yeah makes sense